### PR TITLE
DOC mimevalidator is a part of core

### PR DESCRIPTION
**Pull request by dnsl48**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ and attempts to upload the file.
 
 Install with composer by running `composer require silverstripe/mimevalidator` in the root of your SilverStripe project.
 
+This module is a part of the `silverstripe/recipe-core:^4.6.0`. Thus, it comes out of the box with default silverstripe installations.
+You do not have to install it explicitly, starting with `silverstripe/installer:4.6.0`. An upgrade from the older versions will also install it automatically.
```
Comment (Cheddam) [inline README.md]:
```suggestion
`mimevalidator` is included out of the box via `silverstripe/recipe-core` as of Silverstripe CMS Recipe 4.6.
It is therefore no longer necessary to directly install this module if your project has been upgraded to, or was
created with, CMS 4.6.0 or later.
```

Comment (dnsl48) [reply README.md]: How about
```
Silverstripe CMS Recipe 4.6 and above include this module via `silverstripe/recipe-core`.
Therefore, it is unnecessary to directly install this module if your project has been upgraded to,
or was created with CMS Recipe 4.6.0 or later.
```

**Review by Cheddam — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ and attempts to upload the file.
 
 Install with composer by running `composer require silverstripe/mimevalidator` in the root of your SilverStripe project.
 
+Silverstripe CMS Recipe 4.6 and above include this module via `silverstripe/recipe-core`.
+Therefore, it is unnecessary to directly install this module if your project has been upgraded to,
```
Comment (dhensby) [inline README.md:29]: This isn't quite right - any project that directly uses a module should be declaring it in their own project's `composer.json`. If they aren't directly using it (and just relying on the core's use of it) then this applies

Comment (dnsl48) [reply README.md:29]: Sorry, I'm not sure I'm following. That's now in core, similar to `silverstripe/assets`, for example.
Users don't have to require it explicitly on the application level anymore, if they use CMS Recipes as base for the application.

**Pull request merged**